substrate/frame/glutton/src/benchmarking.rs

```diff
@@ -20,80 +20,121 @@
 //! Has to be compiled and run twice to calibrate on new hardware.
 
 #[cfg(feature = "runtime-benchmarks")]
-use super::*;
-
-use frame_benchmarking::benchmarks;
+use frame_benchmarking::{v2::*, BenchmarkError};
```

Comment:
```suggestion
use frame_benchmarking::v2::*;
```

`BenchmarkError` is already imported in `v2`